src/main/resources/config/liquibase/master.xml

```diff
@@ -129,6 +129,7 @@
     <include file="classpath:config/liquibase/changelog/20210722093000_changelog.xml" relativeToChangelogFile="false"/>
     <include file="classpath:config/liquibase/changelog/20210802213000_changelog.xml" relativeToChangelogFile="false"/>
     <include file="classpath:config/liquibase/changelog/20210812152000_changelog.xml" relativeToChangelogFile="false"/>
+    <include file="classpath:config/liquibase/changelog/20210920225828_changelog.xml" relativeToChangelogFile="false"/>
```

Comment:
wouldn't it be better to put it at the end of the list?

Reply:
There was another changelog at a later date 22nd of September when I merged develop, that is why it is listed not at the end since the long come chronologically :) Shall I still change it?

Reply:
was this PR somewhere deployed before? I think it would make sense to put in last and change the name, i.e. update the date. But this will lead to a mismatch of checksums if the PR was deployed anywhere before.